fft_folder/preparation.py

```diff
@@ -1,22 +1,49 @@
 """
 preparation.py
```

Comment:
Might want to add a little bit about what the file is for or what it is meant to prepare.